docs/developer/load-tests.md

```diff
@@ -0,0 +1,86 @@
+---
+
+# Load and stress tests
+
+TruBudget uses load and stress tests to ensure the system's stability. Some tests are triggered automatically after a major release, others run on each push to the remote repository or get triggered manually. The automated testing strategy allows to regularly check the system minimal requirements and adjust them as needed. Furthermore, developer get detailed information about the system allowing to search for flaws i.e. bottlenecks.
+
+There three types of load/stress tests:
+
+- Stress test: Classic stress to bring the core part of the system (API/Blockchain) to their limits. Although the system capacities are entirely used, the system should not crash nor be unusable
+- Smoke Tests: Smoke test are regular load-test, configured for minimal load. They are used for sanity checks.
+- Stability Test: The whole system and it's interconnecting services such as Email Service and Storage Service are tested. This test, in terms of requests and users per second, is set up similarly to the stress test.
+
+TruBudget uses [k6](https://k6.io) to define and run load and stability tests. Tests can be run both locally and using Docker. We highly recommend using docker since the [provided script](https://github.com/openkfw/TruBudget/tree/master/api/loadtest/runTests.sh) spins up a complete test setup including Grafana and Influx.
+However, if you choose to run the tests locally, `k6` needs to be installed. An optional Grafana and Influx instance are not required, but recommended in order to analyze the data generated by the tests.
+
+:::note
+Per default, the test scripts will run a smoke test against your local TruBudget instance. However, if you want to test a remote instance, the target URL can be changed in `api/loadtest/k6-tests/script.js` by modifying the `BASE_URL` constant.
+The test type can be ganged in the provided script (`api/loadtest/runTests.sh`) by modifying the ` --config` parameter. Valid options are:
+
+- `smoke.config.json`: runs a smoke-test (1 virtual user)
+- `load.config.json`: runs a load-test (100 virtual user)
+- `stress.config.json`: runs a stress-test (400 virtual user)
+
+:::
+
+# Visualizing
+
+To visualize the generate data from the tests via Grafana:
+
+- Open Grafana
+- Navigate to Dashboard
+- In the left sidebar, click the + button > Import
+- Paste the Dashboard ID: 2587 or 11837
+- Click Load
+- Select the Influx data-source
+- Click Import
+
+:::tip
+Grafana has a bunch of pre-configured dashboards for `k6` we recommend using one of the two predefined dashboards (2587 or 11837) mentioned above. However, if you want to browse the available dashboards, you can do it [here](https://grafana.com/grafana/dashboards?search=k6). Also, feel free to contribute to the Grafana community by creating your own dashboard and publishing it to the Grafana Hub.
+:::
+
+# Minimum requirements
+
+TruBudget load and stress tests are run against the minimum and recommend system requirements defied as follows:
```

Comment:
Add "Getting Started" section with a short introduction with commands to start the tests